aip/client-libraries/4222.md

```diff
@@ -154,6 +154,31 @@ If none of the routing parameters matched their respective fields, the routing h
 Much like URL parameters, if there is more than one key-value pair to be sent, the `&`
 character is used as the separator.
 
+### `path_template` syntax
+
+As seen in the above examples, the `path_template` can use a variety of symbols
+that are interpreted by code generators during conversion to regular expression.
+The syntax for these symbols is as follows:
+
+- Wildcards:
+  - `*`: Corresponds to 1 or more non-`/` symbols. The regex describing it is
+    `[^/]+`.
+  - `**`: Corresponds to 0 or more symbols or segments (so it can be empty). The
+    regex is `.*`.
+- `{}`: Named segments, the contents of which are used as a value in the
+  key-value pair of the header value. These have one of the following forms:
+  - `{key}`: Where `key` is the name of the key to be used in the key-value pair
+    sent. Defaults to a format `**` for capturing the contents of the value to
+    be paired.
+  - `{key=format}`: Where `format` is the format of the segment(s) (expressed in
+    this `path_template` syntax) to extract as the value paired with `key`.
+
+If the `path_template` contains a Multivariate Resource ID segment, as defined
```

Comment:
The term "multivariate" doesn't appear in AIP-4231. I believe the term it uses is [Complex resource ID path segments](https://google.aip.dev/client-libraries/4231#complex-resource-id-path-segments).

Reply:
Good point. the design doc used this term, which I copied this text from, but I will change it to align with AIP verbiage

Reply:
Done.